CodeEditModules/Modules/Git/src/Client/Models/ChangedFiles.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  ChangedFiles.swift
+//  
+//
+//  Created by Nanashi Li on 2022/05/05.
+//
+
+import Foundation
+import SwiftUI
+import WorkspaceClient
+
+public struct ChangedFiles: Codable, Hashable, Identifiable {
+    public var id = UUID()
```

Comment:
shouldn't this be a constant?

Reply:
Immutable property will not be decoded because it is declared with an initial value which cannot be overwritten

Reply:
just for the ID

Reply:
make sense! Then we should change everything else to a constant besides the id, thanks!